### DISCUSSION TITLE
Face landmark detection

### DISCUSSION THREAD

**Discussion by nmfisher** [google/flutter-mediapipe#28] [Feature Requests]
Any chance someone has been working on face landmark detection on a local branch somewhere and hasn't pushed yet? I flicked through a few branches and couldn't see anything. Incomplete is fine, I'm happy to work with that too :)

**Comment by charlieforward9** [comment]
It looks like this is on the roadmap and prioritized below the other tasks.

See #13 for the current progress of where this is likely to go.

I am also waiting for this support. It would be incredible to follow the roadmap through a GitHub project if the Mediapipe dev team cared to start managing this repo as such.
